packages/app-project/src/screens/StatsPage/components/ByTheNumbers/ByTheNumbers.js

```diff
@@ -0,0 +1,62 @@
+import counterpart from 'counterpart'
+import { Box, Text } from 'grommet'
+import { number, string } from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+import ContentBox from '../../../../shared/components/ContentBox'
+import Stat from '../../../../shared/components/Stat'
+import AnimatedNumber from '../../../../shared/components/AnimatedNumber'
+
+
+counterpart.registerTranslations('en', en)
+
+function ByTheNumbers (props) {
+  const {
+    classifications,
+    completedSubjects,
+    launchDate,
+    volunteers
+  } = props
+
+  return (
+    <ContentBox title={counterpart('ByTheNumbers.title')}>
+      <Box gap='xsmall'>
+        <Stat
+          value={<AnimatedNumber value={volunteers} />}
+          label={counterpart('ByTheNumbers.volunteers')}
+        />
+        <Stat
+          value={<AnimatedNumber value={classifications} />}
+          label={counterpart('ByTheNumbers.classifications')}
+        />
+        <Stat
+          value={<AnimatedNumber value={completedSubjects} />}
+          label={counterpart('ByTheNumbers.completedSubjects')}
+        />
+        {launchDate && (
+          <Stat
+            value={launchDate}
+            label={counterpart('ByTheNumbers.launchDate')}
+          />
+        )}
+      </Box>
+    </ContentBox>
+  )
+}
+
+ByTheNumbers.propTypes = {
+  classifications: number.isRequired,
+  completedSubjects: number.isRequired,
+  launchDate: string,
+  volunteers: number.isRequired
+}
+
+ByTheNumbers.defaultProps = {
+  classifications: 0,
+  completedSubjects: 0,
+  launchDate: null,
```

Comment:
`null` isn't a string. Should this be an empty string by default?

Reply:
Doesn't really matter, it's not required, so `null` is a permissible value. It can be an empty string if you prefer

Reply:
I was confused because the type is set as a string (I think.) `null` is an object.